src/core/1/guide/guides/kuzzle-depth/scalability/index.md

```diff
@@ -0,0 +1,230 @@
+---
+code: false
+type: page
+title: Scalability
+---
+
+# Scalability
+
+Kuzzle can scale horizontally, provided our [official Cluster Plugin](https://github.com/kuzzleio/kuzzle-plugin-cluster) is installed.
+
+This guide covers how clustering capabilities can be added to Kuzzle.
+
+---
+
+## Quick start
+
+This chapter shows how to quickly create a Kuzzle cluster stack for development purposes. If you already have an existing Kuzzle server running, you may want to read the manual install chapter instead.
+
+<div class="alert alert-info">
```

Comment:
```suggestion
::: info
```

Reply:
Ohhhhhhh NICE :+1: 

Is there a list of these style modifiers somewhere for that new doc version?